.github/workflows/vscode.yml

```
@@ -0,0 +1,51 @@
+    
+    steps:
+      - uses: actions/checkout@v3
+        
+      - name: Update Version
+        run: |
+          cd src/VSCode
+          # Use sed to replace the version in package.json
+          sed -i 's/"version": "[0-9]*\.[0-9]*\.[0-9]*"/"version": "${{ env.VERSION }}"/' package.json
+        
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)